browser/src/ProjectConfig.ts

```diff
@@ -39,13 +39,13 @@ const DefaultConfiguration: IProjectConfiguration = {
  * Search upward for the relevant .oni folder
  */
 export function getProjectConfiguration(filePath: string): Q.Promise<IProjectConfiguration> {
-    const oniDir = findParentDir.sync(filePath, ".oni")
+    const oniDir = findUp.sync(".oni", { cwd: filePath })
```

Comment:
Seems like we could actually make this `async` now (which is the non-blocking code-path). Something like:
```
return findUp(".oni", {  cwd: filePath })
.then((oniDir) => {
      if (!oniDir) {
          return DefaultConfiguration
      }
      return loadConfigurationFromFolder(oniDir)
})
```